tests/run_code_style.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -xeu
+
+if [ $1 = "check" ]; then
```

Comment:
maybe we can move these lines to .github/workflows/ci.yml and remove sh file?

Reply:
Just preserving these scripts for local use for developers, but currently converting those scripts to python scripts to preserve platform agnostic structure.